PrincessBrideTrivia/Program.cs

```diff
@@ -19,12 +20,13 @@ public static void Main(string[] args)
                     numberCorrect++;
                 }
             }
-            Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
-        }
+
+            Console.WriteLine("You got " + $@"{GetPercentCorrect(numberCorrect, questions.Length)}" + " correct");
```

Comment:
```suggestion
            Console.WriteLine("You got " + $"{GetPercentCorrect(numberCorrect, questions.Length)}" + " correct");
```
If you are going to switch it to use interpolation, it would probably be best to ditch all of the concatenations too.